### ISSUE TITLE
Add Visual Spotter Overlay

### ISSUE THREAD

**Issue by samkatakouzinosracing** [SIMRacingApps/SIMRacingApps#214]
Is it possible to add a visual spotter overlay(s)/widgets to show other cars around us without waiting for iRacing to call left/right/three wide?

That’s what car left and car right do but they don't offer positioning indicators.
This video shows what I would like in SimRacingApps.
https://www.youtube.com/watch?v=RNn1VQrp0l8

What I'm asking for would be in addition to the car left right calls widgets.

**Comment by jfgilliam** [comment]
I can't copy what JRT is doing exactly, so let's think of a different way to visualize it. 

Also, there limitations from the telemetry. For example, there's no lat/lon in the telemetry, only the distance from the start/finish line. I use that to position the cars on the track map. So, if a car is approaching you from behind, I cannot tell if it is on your left or right until it actually gets there. Once is gets there and I get the spotter flags saying on which side it is, then I would know. So, just keep that in mind.

I was thinking maybe a vertical window into the track map where the REFERENCE car stays in the center. This would also help on road courses to see the up and coming turn.

**Comment by samkatakouzinosracing** [comment]
> I was thinking maybe a vertical window into the track map

Like looking down at your car from vertically above? 
So Camera is pointing down perpendicular to the ground?

**Comment by samkatakouzinosracing** [comment]
I would like to see vehicle markers having color schemes similar to the iRacing relatives box.
One color palette to indicate vehicle class.
One color palette to indicate lap relative to you (same lap, lap ahead, lap behind).

**Comment by jfgilliam** [comment]
Yes, looking down from above. The track map has the iRacing colors. I would do the same for this.

**Comment by samkatakouzinosracing** [comment]
I would like vehicle markers showing for vehicles ahead of me, as well as behind me.
With other systems I only know of what's behind and beside me.

A lot of times another vehicle is ahead of me yet outside my field of view.
This happens a lot in oval pack racing and a few times in road racing. 
This with a single monitor.

It would give me another layer of confidence to keep my eyes forward and 'see' what the other car is doing near me.
Are they pulling away? Are they still near my front quarter but not close enough to get called by iRacing? Am I closing in on them?
So many times I back out of a competitive move because the other vehicle is in my blind spot ahead of me (not called by iRacing and not seen within my field of view)
